site/content/en/docs/admission-check-controllers/multikueue.md

```diff
@@ -0,0 +1,85 @@
+---
+
+The MultiKueue Admission Check Controller is an Admission Check Controller designed to provide the manager side functionality for multi cluster job dispatching.
+
+You can enable it by setting the `MultiKueue` feature gate. Check the [Installation](/docs/installation/#change-the-feature-gates-configuration) guide for details on feature gate configuration.
+
+{{% alert title="Warning" color="warning" %}}
+MultiKueue is currently an alpha feature and disabled by default.
+{{% /alert %}}
+
+## MultiKueue Overview
+
+A MultiKueue setup is composed of a manager cluster and at least one worker cluster.
+
+### Cluster Roles
+#### Manager Cluster
+
+The manager's main attributes are:
+- Establish and maintain the connection with the worker clusters.
+- Create and monitor remote objects (workloads or jobs) while keeping the local ones in sync.
+
+The MultiKueue Admission Check Controller runs in the manager cluster and will also maintain the `Active` status of the Admission Checks controlled by `multikueue`.
+
+The ResourceQuota set on a ClusterQueue using MultiKueue controls how many jobs are subject for dispatching at a given point in time, ideally it should be less then the total quotas in the worker clusters since otherwise will cause the monitoring of workloads that don't have a chance to be admitted. 
+
+#### Worker Cluster
+
+The worker cluster acts like an usual standalone Kueue cluster in which the Kueue the workloads and jobs are managed by the MultiKueue Admission Check Controller running in the manager cluster.
+
+### Job Flow
+
+If the ClusterQueue a job is assigned to in the manager cluster uses a MultiKueue AdmissionCheck it will be the subject of dispatching, as a result:
+
+- When it's workload gets a QuotaReservation in the manager, a copy of that workload will be created in all the configured worker clusters.
+- When one of the remote workloads is admitted in a worker cluster:
+  - All the other remote workloads are removed.
+  - A copy of the job is created in the selected worker cluster having the remote workload set as its prebuilt-workload.
+  - The AdmissionCheckState of the local workload is set as `Ready` resulting in its Admission in the manager cluster.
+- The remote objects , workload and job, are monitored and the changes in their status are synced back in the manager objects.
+- When the remote workload is marked as `Finished`:
+  - A last sync is done for the objects status.
+  - The remote objects are removed from the worker cluster.
+
+
+## Supported jobs
+### batch/Job
+Known Limitations:
+- Since unsuspending a Job in the manager cluster will lead to its local execution, the AdmissionCheckStates are kept `Pending` during the remote job execution.
+- Since updating the status of a local Job could conflict with the Job's main controller, the Job status is not synced during the job execution, the final status of the remote Job is copied when the remote workload is marked as `Finished`.
```

Comment:
```suggestion
- Since updating the status of a local Job could conflict with the Kubernetes Job controller, the manager
  does not sync the Job status during the job execution.
  The manager copies the final status of the remote Job when the remote workload is marked as `Finished`.
```

Reply:
done